### PR TITLE
Added support for the word "decade".

### PR DESCRIPTION
**Pull request by seric17**
For issue #725 .

### CODE REVIEW

**Comment by Gallaecio** [comment]
It looks like tests did not run, I see “This branch cannot be rebased due to conflicts” here in Github. Could you rebase against `master`?

**Comment by seric17** [comment]
I think I rebased it correctly. Please let me know if it works now.

File: dateparser/data/date_translation_data/en.py [hunk 1]
```diff
@@ -252,6 +259,14 @@
             "(\\d+) second ago",
             "(\\d+) seconds ago",
             "(\\d+) sec ago"
+        ],
```
Comment (noviluni) [inline dateparser/data/date_translation_data/en.py:262]: don't forget to update this file with `write_complete_data.py` when updating the `yaml` files.

**Comment by seric17** [comment]
Thanks for the feedback. I tried to add all the suggested changes. Please let me know if anything is still not working.

**Comment by noviluni** [comment]
Thank you @seric17!

**Pull request merged**